Gopkg.toml

```diff
@@ -14,42 +14,33 @@ required = ["github.com/docker/distribution"]
 
 [[constraint]]
   name = "github.com/ti-mo/conntrack"
-  revision = "309d2d6a30782ae8f0c68f2dda0774966523e469"
-  source = "https://github.com/aporeto-inc/conntrack.git"
-
-[[constraint]]
-  name = "k8s.io/api"
-  version = "kubernetes-1.13.1"
-
-[[constraint]]
-  name = "k8s.io/client-go"
-  version = "kubernetes-1.13.1"
-
-[[constraint]]
-  branch = "release-3.13.0"
-  name = "github.com/aporeto-inc/oxy"
-
-[[constraint]]
-  branch = "release-3.13.0"
-  name = "go.aporeto.io/netlink-go"
-
-[[constraint]]
-  name = "github.com/hashicorp/go-version"
-  version = "v1.0.0"
+  branch = "master"
```

Comment:
@mheese my only thought is : do we want github.com/ti-mo/conntrack:**master** ?
I think it might be better to pin it to the current master revision @ sha c9b176489c1a057922cbaa0a78df9e51cffbcc0d

```
  name = "github.com/ti-mo/conntrack"
  revision = "c9b176489c1a057922cbaa0a78df9e51cffbcc0d"
  source = "https://github.com/ti-mo/conntrack.git"
```

Reply:
there is not even a constraint in general for this anymore, that's why I just put this to master